extensions/healthie/actions/getMostRecentMetricEntry/getMostRecentMetricEntry.test.ts

```diff
@@ -0,0 +1,38 @@
+import { generateTestPayload } from '../../../../src/tests'
+import { getSdk } from '../../gql/sdk'
+import { mockGetSdk } from '../../gql/__mocks__/sdk'
+import { getMostRecentMetricEntry } from '../getMostRecentMetricEntry'
+
+jest.mock('../../gql/sdk')
+jest.mock('../../graphqlClient')
+
+describe('createMetricEntry action', () => {
+  const onComplete = jest.fn()
+  const onError = jest.fn()
+
+  beforeAll(() => {
+    ;(getSdk as jest.Mock).mockImplementation(mockGetSdk)
+  })
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('Should get most recent metric entry', async () => {
+    await getMostRecentMetricEntry.onActivityCreated(
+      generateTestPayload({
+        fields: {
+          category: 'Weight',
+        },
+        settings: {
+          apiKey: 'api-key',
+          apiUrl: 'api-url',
+        },
+      }),
+      onComplete,
+      onError
+    )
+    expect(onComplete).toBeCalledTimes(1)
```

Comment:
I would prefer a `toBeCalledWith` assertion to test explicitly the value we are storing in the data points:

```javascript
    expect(onComplete).toBeCalledWith({
      data_points: {
        metricId: '...',
        lastMetricValue: 1,
        metricDate: '...'
      },
    })
```